smart_contracts.rst

```diff
@@ -262,10 +262,11 @@ This contract creates and remembers a TokenNetwork contract for an ERC20 Token.
 
 Attributes:
 
-- ``address public secret_registry_address``
-- ``uint256 public chain_id``
-- ``uint256 public settlement_timeout_min``
-- ``uint256 public settlement_timeout_max``
+- ``address public secret_registry_address`` The SecretRegistry to be used by all TokenNetworks in this registry.
+- ``uint256 public chain_id`` The chain ID of the blockchain where this contract is deployed.
+- ``uint256 public settlement_timeout_min`` The longest settlement period that channel openers can choose (in number of blocks).
```

Comment:
You mixed up shortest and longest in these w´two comments.

Reply:
I'll fix it.